datalad/distribution/create_publication_target_sshwebserver.py

```diff
@@ -154,16 +153,15 @@ def __call__(sshurl, target=None, target_dir=None,
         assert(ds.repo is not None)
 
         # determine target parameters:
-        parsed_target = urlparse(sshurl)
-        host_name = parsed_target.netloc
+        sshri = RI(sshurl)
 
-        # TODO: Sufficient to fail on this condition?
-        if not parsed_target.netloc:
-            raise ValueError("Malformed URL: {0}".format(sshurl))
+        if not isinstance(sshri, SSHRI) \
+                and not (isinstance(sshri, URL) and sshri.scheme == 'ssh'):
+                    raise ValueError("Unsupported SSH URL: '{0}', use ssh://host/path or host:path syntax".format(sshurl))
```

Comment:
confused by failed tests on travis :
```
======================================================================

ERROR: datalad.support.tests.test_sshconnector.test_ssh_get_connection

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/nose/case.py", line 197, in runTest

    self.test(*self.arg)

  File "/home/travis/build/datalad/datalad/datalad/tests/utils.py", line 765, in newfunc

    return func(*args, **kwargs)

  File "/home/travis/build/datalad/datalad/datalad/support/tests/test_sshconnector.py", line 33, in test_ssh_get_connection

    assert_raises(ValueError, manager.get_connection, 'localhost')

  File "/opt/python/2.7.9/lib/python2.7/unittest/case.py", line 473, in assertRaises

    callableObj(*args, **kwargs)

  File "/home/travis/build/datalad/datalad/datalad/support/sshconnector.py", line 154, in get_connection

    raise ValueError("Unsupported SSH URL: '{0}', use ssh://host/path or host:path syntax".format(sshurl))

NameError: global name 'sshurl' is not defined
```
sshurl is present there... I guess running tests locally with that env var would help